nservicebus/upgrades/supported-versions.js

```diff
@@ -0,0 +1,23 @@
+var now = new Date();
```

Comment:
Could you please wrap this whole thing in an immediately-executed function expression (IIFE) so we don't further pollute the global namespace?

```
'use strict';
(function () {
    // All your code
} ());